Source/santactl/Commands/SNTCommandSync.m

```diff
@@ -51,6 +47,7 @@ + (NSString *)longHelpText {
   return (@"If Santa is configured to synchronize with a server, "
           @"this is the command used for syncing.\n\n"
           @"Options:\n"
+          @"  --new: Perform a new sync after an already in progress sync.\n"
```

Comment:
Is there much of a need for such a feature? If someone is concerned that the currently running sync isn't good enough for some reason, I think it's fine to just require them to run the command again in a few minutes (which is effectively all this flag seems to do anyways).